ipc/socket/socket_factory.go

```diff
@@ -189,7 +189,7 @@ func (f *Factory) findRelayWithContext(ctx context.Context, w *worker.Process) (
 	}
 }
 
-func (f *Factory) findRelay(w *worker.Process) (*socket.Relay, error) {
+func (f *Factory) findRelay(w *worker.Process) (*socket.Relay, error) { //nolint:unused
```

Comment:
Delete it if it's unused :)

Reply:
Could this method be used by someone in their plugins?

Reply:
It's private :)

Reply:
damn